README.md

```diff
@@ -41,34 +38,100 @@ require('apostrophe')({
+
+>One caveat of sharing documents between sites is that the modules for page-types and piece-types must be the same in each. For example, if site A is using a page-type of `contact-page`, a page of that type can only be imported into a site that also has a `contact-page` module. If the codebase of the two projects are significantly different, it can either cause the import to fail, or some data to be lost if schema fields can't be reconciled.
+
+### Permissions
+
+Any user that can publish a document can export and import documents of that same type. Users who can only create or modify documents of a specific type will not be import documents, but they will be able to export documents of that same type.
 
 ### Options
 
-You can disable the export and/or the import for certain pieces this way:
+You can disable the export and/or the import for any page- or piece-type using the `importExport` option. This option takes an object with `import` and `export` keys that take both can take boolean values.
 
 ```javascript
 module.exports = {
-  extends: '@apostrophecms/piece-type'
+  extend: '@apostrophecms/piece-type',
   options: {
     importExport: {
-      import: false,
+      import: true,
       export: false
-    }
-  }
-}
+    },
+    label: 'Article',
+    pluralLabel: 'Articles'
 ```
 
-### Export
-
-You can export documents from batch operations as well as from context operations. 
-Pages can only be exported from context operations since they don't support batch operations.
-Currenlty it only exports the `.tar.gz` format.
-
-
-### Import
+In this example, the 'Article' piece-type can be imported from a file, but users won't be allowed to export any of this type. This will also cause the export option to disappear from both the batch and individual 'Article' piece context menus in the 'Article' piece-type manager. Note that this will not impact other piece-types. The removal of the export menus from the respective managers will also occur if page-type or document templates are disabled for exporting. Finally, any disabled document type will not appear on the list of related documents.
 
-You can import **pieces** and **pages** from utility operations.
+Disabling a document type from being imported will remove the utility context menu from the respective document manager. It will also block insertion of any documents of that type when importing from the context menu of another manager. Finally, any export file that contains a related document type that is disabled for import, will still successfully import for those document types that are allowed.
```

Comment:
Will it necessarily remove that menu? There could be other utility operations in the project. I would skip saying that.

Reply:
"import, will" -> lose the comma I think

Reply:
Reworded and deleted.